src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+    protected String by;
```

Comment:
Is using protected the most suitable option ? It makes it accessible to other classes from the same package ... I noticed the same issue in several other places too.